Sources/Kodable/Codable+Lossless.swift

```diff
@@ -6,7 +6,7 @@ public typealias LosslessDecodable = LosslessStringConvertible & Decodable
 
 // MARK: Helper type to decode lossless values
 
-struct LosslessValue<T: LosslessDecodable>: Decodable {
+struct LosslessValue<T: Decodable>: Decodable {
```

Comment:
Is this change required ? given this `guard let anyLosslessDecodable = T.self as? LosslessDecodable.Type`